From f9084deed250c40f6d64e489441995c8b388dfe1 Mon Sep 17 00:00:00 2001
Message-ID: <f9084deed250c40f6d64e489441995c8b388dfe1.1707944340.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1707944340.git.sharlatanus@gmail.com>
References: <cover.1707944340.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 14 Feb 2024 20:45:56 +0000
Subject: [PATCH 3/6] gnu: Add go-github-com-bmatcuk-doublestar-v2.

* gnu/packages/golang.scm (go-github-com-bmatcuk-doublestar-v2): New variable.

Change-Id: I56d5588a1741a8e6c86320ec07bc86805f34fc91
---
 gnu/packages/golang.scm | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 25e9715c79..87ee0d21d2 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -7545,6 +7545,27 @@ (define-public go-github-com-bmatcuk-doublestar
 matching and globbing with support for \"doublestar\" patterns.")
     (license license:expat)))
 
+;; For chezmoi-1.8.10
+(define-public go-github-com-bmatcuk-doublestar-v2
+  (package
+    (inherit go-github-com-bmatcuk-doublestar)
+    (name "go-github-com-bmatcuk-doublestar-v2")
+    (version "2.0.4")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/bmatcuk/doublestar")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "09ma8a9rhs8dg527vjhdf3lsb6lajaq193m6ksssm2k3qajhpi94"))))
+    (arguments
+     (list
+      #:tests? #f ; tests have more broken parts
+      #:unpack-path "github.com/bmatcuk/doublestar/v2"
+      #:import-path "github.com/bmatcuk/doublestar/v2"))))
+
 (define-public go-github-com-dlclark-regexp2
   (package
     (name "go-github-com-dlclark-regexp2")
-- 
2.41.0

